common/ferrostar/src/navigation_controller/mod.rs

```diff
@@ -405,6 +438,20 @@ impl NavigationController {
     }
 }
 
+impl Navigator for NavigationController {
+    fn get_initial_state(&self, location: UserLocation) -> TripState {
+        self.get_initial_state(location)
+    }
+
+    fn advance_to_next_step(&self, state: &TripState) -> TripState {
+        self.advance_to_next_step(state)
+    }
+
+    fn update_user_location(&self, location: UserLocation, state: &TripState) -> TripState {
+        self.update_user_location(location, state)
+    }
+}
```

Comment:
Rather than making methods that just call other methods by the same name, moving the implementation under the trait is more idiomatic.

(I'm actually surprised that this is legal; the signatures are the same, but yeah Rust traits can get a bit weird...)